webview/src/plots/components/comparisonTable/ComparisonTableHead.tsx

```diff
@@ -28,6 +28,26 @@ export const ComparisonTableHead: React.FC<ComparisonTableHeadProps> = ({
     (state: PlotsState) => state.dragAndDrop.draggedRef?.itemId
   )
   const ribbonHeight = useSelector((state: PlotsState) => state.ribbon.height)
+  const [top, setTop] = useState(0)
+  const headRef = createRef<HTMLTableSectionElement>()
+
+  useEffect(() => {
+    const calculateTop = () => {
```

Comment:
We do need sticky headers for the revisions. I am also seeing some jank:

https://user-images.githubusercontent.com/37993418/226214117-9344ba26-ed87-4739-9d01-e3b51af1cf7c.mov

Note: There is a visible gap between the normal sticky headers and the ribbon in the video.

Reply:
> Maybe it's just on my end (or I just scroll more aggressively 😄), but the image headers look really buggy...
> 


> Should we consider not making the image headers sticky at all?

Went back and tried making this faster. Decided to avoid React as much as possible for the style update. `useState` is a little too efficient with reconciliation and sometimes makes things a little less smooth. It's is not as perfect as it would with CSS only, but it's way much better now.

https://user-images.githubusercontent.com/3683420/226367635-ddc7c30e-6462-4c12-be83-49140611d44a.mov

Reply:
> We do need sticky headers for the revisions. I am also seeing some jank:
> 
> Note: There is a visible gap between the normal sticky headers and the ribbon in the video.

I am not able to reproduce. Might have to do with resolution and unfortunately I cannot test on a bigger screen at the moment. Can you check `PlotsContainer.tsx` on line 138 (`stickyHeaderTop={ribbonHeight - 4}`) and play with a higher than 4 value and tell me which value would close that gap for you? Removing a few pixels from the header won't be that apparent for other users, but if it closes the gap for users like you that'd would make it perfect.